tests/queries/0_stateless/03146_asof_join_ddb_merge_long.sql.j2

```diff
@@ -0,0 +1,39 @@
+-- Tags: long
+
+SET allow_experimental_analyzer=1;
+
+SET session_timezone = 'UTC';
+
+{% for join_algorithm in ['default', 'full_sorting_merge'] -%}
+
+SET join_algorithm = '{{ join_algorithm }}';
+
+-- TODO: support enable for full_sorting_merge
```

Comment:
It's disabled not because ASOF, but because of USING and `join_use_nulls` is not supported in `full_sorting_merge`

https://github.com/ClickHouse/ClickHouse/blob/fbb7a3ed5201e11b4a638e77a1a149a51792ed00/src/Interpreters/FullSortingMergeJoin.h#L70-L71


```suggestion
-- TODO: enable once USING and `join_use_nulls` is supported by `full_sorting_merge`
```